### DISCUSSION TITLE
执行build.sh报错

### DISCUSSION THREAD

**Discussion by diaowentao** [apache/doris#7704] [A - General / Q&A]
Build Frontend Modules: fe-common,spark-dpp,fe-core
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.2.0.RELEASE/spring-boot-starter-parent-2.2.0.RELEASE.pom
Downloading from cloudera-public: https://repository.cloudera.com/artifactory/public/org/springframework/boot/spring-boot-starter-parent/2.2.0.RELEASE/spring-boot-starter-parent-2.2.0.RELEASE.pom
Downloading from oracleReleases: http://download.oracle.com/maven/org/springframework/boot/spring-boot-starter-parent/2.2.0.RELEASE/spring-boot-starter-parent-2.2.0.RELEASE.pom
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.apache:doris-fe:3.4.0: Could not transfer artifact org.springframework.boot:spring-boot-starter-parent:pom:2.2.0.RELEASE from/to central (https://repo.maven.apache.org/maven2): Connect to repo.maven.apache.org:443 [repo.maven.apache.org/151.101.24.215] failed: Connection refused (Connection refused) and 'parent.relativePath' points at wrong local POM @ line 40, column 13
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.apache:doris-fe:3.4.0 (/opt/doris/fe/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.apache:doris-fe:3.4.0: Could not transfer artifact org.springframework.boot:spring-boot-starter-parent:pom:2.2.0.RELEASE from/to central (https://repo.maven.apache.org/maven2): Connect to repo.maven.apache.org:443 [repo.maven.apache.org/151.101.24.215] failed: Connection refused (Connection refused) and 'parent.relativePath' points at wrong local POM @ line 40, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

**Comment by BiteTheDDDDt** [comment]
换个maven源试试？
